### ISSUE TITLE
Damage Overlay bugged

### ISSUE THREAD

**Issue by Alf21** [TTTReborn/tttreborn#204]
## Prerequisites

- [x] I have searched [existing bug reports](https://github.com/TTTReborn/ttt-reborn/issues?q=label%3Aissue%2Fbug-report) and confirmed my issue is not already reported.

## Summary
Some s&box update broke the damage overlay. Opacity with background image interaction seems to be bugged.

## Steps to reproduce
Just receive damage in-game. The damage overlay should look like sh*t.

**Comment by mzegar** [comment]
@Krzyska-Filip do you want to throw up a PR for this? Thanks for looking into this.

**Comment by GravityWolfNotAmused** [comment]
Can this be closed?

**Comment by Krzyska-Filip** [comment]
(after my PR) 
Damage indicator had minor bug where executing suicide while being dead would display damage indicator. Alternative was that last dmg before death was not displaying properly.

(after recent AliveHud PR / I haven't checked it)
Damage indicator will dissapear after death which will look a bit odd (a least for me)

(Ideal behaviour)
Damage indicator display after death but does not trigger when player is dead. Possible, it would mean that we have to move damage indicator from AliveHUD to GeneralHUD and implement PlayerState.Dying unless someone know better way to solve it.

**Comment by mzegar** [comment]
Yeah that should fix the issue, let's keep this GitHub issue for this as it's tangentially related.
